### PR TITLE
data:YuvrajChhetri

### PR DESCRIPTION
**Pull request by i-am-yuvi**

### CODE REVIEW

File: data/YuvrajChhetri.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/yuvraj-chhetri-9ba17018a/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/i-am-yuvi",
```
Comment (ChinmayMhatre) [inline data/YuvrajChhetri.json:20]: Hey @i-am-yuvi , you'll have to rename the file with your github username. i.e i-am-yuvi.json.

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

File: data/YuvrajChhetri.json (hunk 1 above)
Comment (i-am-yuvi) [reply data/YuvrajChhetri.json:20]: Sure i'll do that

**Comment by ChinmayMhatre** [comment]
LGTM!

**Review by dan-mba — APPROVED** [review]
Looks good to me

**Pull request merged**